ironic.conf.j2

```diff
@@ -71,23 +117,19 @@ pxe_config_template = $pybasedir/drivers/modules/ipxe_config.template
 tftp_master_path = /shared/tftpboot
 tftp_root = /shared/tftpboot
 uefi_pxe_config_template = $pybasedir/drivers/modules/ipxe_config.template
+pxe_append_params = nofb nomodeset vga=normal ipa-insecure=1
 
 [redfish]
 use_swift = false
 
-[ipmi]
-# use_ipmitool_retries transfers the responsibility of retrying to ipmitool
-# when supported. If set to false, then ipmitool is called as follows :
-#    $ipmitool -R 1 -N 1 ...
-# and Ironic handles the retry loop.
-use_ipmitool_retries = true
-# The following parameters are the defaults in Ironic. They are used in the
-# following way if use_ipmitool_retries is set to true:
-#    $ipmitool -R <X> -N <Y> ...
-# where :
-#    X = command_retry_timeout / min_command_interval
-#    Y = min_command_interval
-# If use_ipmitool_retries is false, then ironic retries X times, with an
-# interval of Y in between each tries.
-min_command_interval = 5
-command_retry_timeout = 60
+[service_catalog]
+endpoint_override = {{ env["IRONIC_BASE_URL"] }}:6385
+{% if "CACERT_FILE" in env %}
+cafile = {{ env["CACERT_FILE"] }}
+{% endif %}
+
+{% if "CERT_FILE" in env %}
+[ssl]
```

Comment:
Just a note. The built-in TLS support works, but may not be the most robust in the world. Ideally, we should use something like nginx for that. Likely not in the first iteration though.